src/lib/CRSF/CRSF.h

```diff
@@ -80,7 +80,7 @@ class CRSF
     static void GetMspMessage(uint8_t **data, uint8_t *len);
     static void UnlockMspMessage();
     static void AddMspMessage(const uint8_t length, uint8_t* data);
-    static void AddMspMessage(mspPacket_t* packet);
+    static void AddMspMessage(mspPacket_t* packet, uint8_t destination = CRSF_ADDRESS_FLIGHT_CONTROLLER);
```

Comment:
I would prefer to not use a default parameter, especially for the destination. I think it would make everywhere clearer if CRSF::AddMspMessage included the intended destination, especially as we move more toward actually using the SRC/DEST fields properly.